src/status.rs

```diff
@@ -3,7 +3,7 @@ use std::fmt;
 use h2;
 use http::header::HeaderValue;
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
While adding derived traits here seems pretty harmless, I am a little concerned that it could potentially break backwards compatibility in the future, if we add fields to `Status` that can't implement `Eq` or `PartialEq`. Also, since the purpose of this branch is to add tests, I'd prefer it to not also make this change. Can we rewrite the test to use something like this?
```rust
if status.code() == Status::OK.code() {
    // ...
}
```

Reply:
I agree that `Eq` and `PartialEq` should be punted to `code()`. Especially after #78 `Status` shouldn't impl these traits.